docs/source/en/intro/quickstart.md

```diff
@@ -408,6 +408,7 @@ with the above short sections.
 We recommend that developers continue reading other documents:
 - Egg provides a powerful mechanism for extending features. See [Plug-ins](../advanced/plugin.md).
 - As large teams need to follow certain constraints and conventions, in egg, we recommend further creating an upper-level framework that fits your team. See [Frameworks](../advanced/framework.md).
+- Egg is a progressive framework, reuse of code is painless. See [Progressive](../tutorials/progressive.md).
```

Comment:
any suggest for `- Egg is a progressive framework, reuse of code is painless. See [Progressive](../tutorials/progressive.md).` ?